language/sygus.tex

```diff
@@ -293,6 +298,13 @@ \subsubsection{Changes from SyGuS 2.0 to SyGuS 2.1}
 \item Oracle Functions are introduced. These are functional symbols whose interpretation is associated to an external oracle. The functions are treated as universally quantified uninterpreted functions, and calls to that external oracle place with concrete input values generate assumptions over the behaviour of the uninterpreted function. These assumptions are conjoined to the list of assumptions $\alpha$ in the synthesis conjecture.
 
 \item A set of new commands are introduced to enable easy specification of specific types of oracles, for instance input-output oracles. These oracles are based on the formalization by Jha and Seshia~\cite{oracles}.
+
+\item A new SyGuS logic, called \sygusinline|CHC_$X$|,
+has been introduced for specifying synthesis problems using constrained Horn clause systems
+over an SMT-LIB base logic $X$.
```

Comment:
Can you also mentioned here that a new command \constraintchckwd is introduced.

Reply:
Oh yes, good catch! Thanks.